### ISSUE TITLE
Pkg.add() very slow / hangs on Windows

### ISSUE THREAD

**Issue by marcpabst** [JuliaLang/julia#27585]
Having worked with R for some time I wanted to try Julia because it seems much more sleek and overall more appealing. 

But having a fresh Julia installation here, Pkg.init() takes at least 20 minutes to complete `INFO: Cloning METADATA from https://github.com/JuliaLang/METADATA.jl`. After that, Pkg.add("Atom") will just hang with no output at all for hours. This is actually the same for all packages names, including non-existing ones.

I'm using Version 0.6.3 on Win10 (x86_64-w64-mingw32).

**Comment by marcpabst** [comment]
Well, this is a real problem for me. Any way to use the new package manager without installing a nightly build?

**Comment by StefanKarpinski** [comment]
Yes, I'm sorry about the slowness on Windows. There is a 0.7.0-alpha release you can use.
